### PR TITLE
Changed occurrences of Myriad and Hadoop versions

### PR DESCRIPTION
**Pull request by smarella**
Myriad: 0.0.1 -> 0.1.0
Hadoop: 2.7.1 -> 2.7.0

### CODE REVIEW

**Comment by sdaingade** [comment]
Looks good to me :rocket:

**Comment by kensipe** [comment]
LGTM!

File: myriad-scheduler/src/main/java/org/apache/myriad/configuration/MyriadConfiguration.java [hunk 1]
```diff
@@ -52,7 +52,7 @@
  * cgroups: false
  * executor:
  * jvmMaxMemoryMB: 256
- * path: file://localhost/usr/local/libexec/mesos/myriad-executor-runnable-0.0.1.jar
+ * path: file://localhost/usr/local/libexec/mesos/myriad-executor-runnable-0.1.0.jar
```
Comment (DarinJ) [inline myriad-scheduler/src/main/java/org/apache/myriad/configuration/MyriadConfiguration.java]: can probably just remove this line as it's no longer needed.

**Comment by DarinJ** [comment]
LGTM, the comment is minor fix it later.

**Pull request closed without merging**